internal/abi/fastly/hostcalls_guest.go

```diff
@@ -1390,50 +1401,8 @@ func (r *PendingRequest) Wait() (response *HTTPResponse, responseBody *HTTPBody,
 //	   (result $resp_body $body_handle)
 //	)
 //
-//go:wasm-module fastly_http_req
-//export pending_req_select
+//go:wasmimport fastly_http_req pending_req_select
 //go:noescape
-func fastlyHTTPReqPendingReqSelect(
-	hs []pendingRequestHandle, // TODO(pb): is correct?
-	doneIdx *prim.U32,
-	resp *responseHandle,
-	respBody *bodyHandle,
-) FastlyStatus
-
-// PendingRequestSelect blocks until one of the provided pending requests is
-// complete. Returns the completed request, and its associated response and
-// response body. If more than one pending request is complete, returns one of
-// them randomly.
-//
-// TODO(pb): is random correct?
-func PendingRequestSelect(reqs ...*PendingRequest) (index int, done *PendingRequest, response *HTTPResponse, responseBody *HTTPBody, err error) {
```

Comment:
If the intent is to fully drop this function, can you remove it from `hostcalls_noguest.go` too?